Code/GraphMol/DistGeomHelpers/Embedder.h

```diff
@@ -221,6 +221,70 @@ INT_VECT EmbedMultipleConfs(
     bool enforceChirality = true, bool useExpTorsionAnglePrefs = false,
     bool useBasicKnowledge = false, bool verbose = false,
     double basinThresh = 5.0);
+
+struct EmbedParameters {
+  unsigned int maxIterations;
+  int numThreads;
+  int randomSeed;
+  bool clearConfs;
+  bool useRandomCoords;
+  double boxSizeMult;
+  bool randNegEig;
+  unsigned int numZeroFail;
+  const std::map<int, RDGeom::Point3D> *coordMap;
+  double optimizerForceTol;
+  bool ignoreSmoothingFailures;
+  bool enforceChirality;
+  bool useExpTorsionAnglePrefs;
+  bool useBasicKnowledge;
+  bool verbose;
+  double basinThresh;
+  double pruneRmsThresh;
+  EmbedParameters()
+      : maxIterations(0),
+        numThreads(1),
+        randomSeed(-1),
+        clearConfs(true),
+        useRandomCoords(false),
+        boxSizeMult(2.0),
+        randNegEig(true),
+        numZeroFail(1),
+        coordMap(NULL),
+        optimizerForceTol(1e-3),
+        ignoreSmoothingFailures(false),
+        enforceChirality(true),
+        useExpTorsionAnglePrefs(false),
+        useBasicKnowledge(false),
+        verbose(false),
+        basinThresh(5.0),
+        pruneRmsThresh(-1.0){};
+};
+inline int EmbedMolecule(ROMol &mol, const EmbedParameters &params) {
```

Comment:
An awesome thing you can do with structs is make common settings for standard ops.

i.e.

```
const EmbedParameters ETDKG = {.....};
```

So people like me with bad memories have a reference.